stdlib/LibGit2/src/commit.jl

```diff
@@ -147,3 +147,45 @@ function commit(repo::GitRepo, msg::AbstractString;
     end
     return commit_id
 end
+
+"""
+    parentcount(c::GitCommit)
+
+Get the number of parents of this commit.
+
+See also [`parent`](@ref), [`parent_id`](@ref).
+"""
+parentcount(c::GitCommit) =
+    Int(ccall((:git_commit_parentcount, :libgit2), Cuint, (Ptr{Cvoid},), c))
```

Comment:
I know this isn't new in this PR, but why is this entire stdlib hardcoding the libname rather than using the jll? That seems like a problem